### ISSUE TITLE
[Accessibility][Product Carousel] The "previous and "next" buttons for the carousel have insufficient color contrast.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1763]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Test color contrast with the Accessible Color Picker tool.

**Current Behavior**
The "previous" and "next" buttons are gray (#AAA) on a white (#FFF) background, resulting in an insufficient color contrast of 2.32:1. Users with low vision will have difficulty distinguishing these controls.

**Expected behavior/code**
Interactive elements need to meet a minimum color contrast ratio of 3:1. Designers and developers need to use colors that meet this minimum.

**Environment**
Chrome 91, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision

**WCAG SC**
1.4.11 Non-text Contrast

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
